_posts/2020-04-16-remote-junior-developer-tips.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: 'Tips From One Remote Junior Developer to Another.'
+date: 2020-04-19 10:30:00
+categories: ["junior", "remote"]
+author: fionaDL
```

Comment:
@FionaDL In order for this to work, you should add you at the end of this file: https://github.com/ombulabs/blog/blob/be480848a601f18c4eb04f01ce2a0091755fbdaf/_config.yml

For the avatar, you'll need to signup in https://en.gravatar.com/ , upload your photo, and then put the id in the `_config.yml` file.